charts/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v2
-version: 4.3.0-develop
+version: 4.3.0
 name: openebs
-appVersion: 4.3.0-develop
+appVersion: 4.3.0
```

Comment:
Why are we reverting this? The develop chart should always be vNext-develop

Reply:
I was testing with these changes. I've reverted this change.